dist/server/WebsiteRouter.js

```diff
@@ -0,0 +1,230 @@
+//////////////////////////////////////////////////
+// Silex, live web creation
+// http://projects.silexlabs.org/?/silex/
+//
+// Copyright (c) 2012 Silex Labs
+// http://www.silexlabs.org/
+//
+// Silex is available under the GPL license
+// http://www.silexlabs.org/silex/silex-licensing/
+//////////////////////////////////////////////////
+
+const Path = require('path');
+const express = require('express');
+const fs = require('fs');
+const { JSDOM } = require('jsdom');
+const bodyParser = require('body-parser');
```

Comment:
As seen in CE2, you don't need `body-parser` since `express 4.16.0`

Reply:
Concerning JSON: CE adds routes that collide with Silex new routes, e.g. `/website/github/get/master/` will be caught by CE as a service called "website/github". So I need to put Silex router before CE router, and so the body needs to be explicit

Concerning text: here I need the body parsed as text, and I can't figure out how to access the body parser from express to use bodyParser.text(

Also in the docs, this is still written to require('body-parser') so I lost too much time with this for now
https://expressjs.com/en/4x/api.html#req.body

Reply:
finally I removed it from CE2 since it doesn't need a parser (right?)
and I added it here https://github.com/silexlabs/Silex/pull/718

Reply:
I think the documentation is outdated on this point but the changelog clearly mentioned the new way to parse bodies https://expressjs.com/en/changelog/4x.html